react/components/Image.js

```diff
@@ -4,18 +4,46 @@ import PropTypes from 'prop-types'
 import productSummary from '../productSummary.css'
 
 /** Image component with 1:1 aspect ratio */
-export default function Image({ src, className }) {
+export default function Image({ src, className, width, height }) {
+  const resizeSrc = src => {
+    //Check of src is already in a size defined by the client
+    const isWithMeasures = new RegExp(/[1-9]+-(:?([1-9]+)|(auto))/);
+    if (isWithMeasures.exec(src)) {
+      return src
+    }
+
+    const imageId = src.slice(src.indexOf("/ids/")).split("/")[2]
+    const withSuffix = `${imageId}-${width}-${height}`
+
+    return src.replace(imageId, withSuffix)
+  }
+
   return (
     <div
       className={`${className} ${productSummary.aspectRatio} w-100`}
-      style={{ backgroundImage: `url(${src})` }}
+      style={{ backgroundImage: `url(${resizeSrc(src)})` }}
```

Comment:
The PR title says that now the `Img` tag will be used. Wouldn't it change something here?

Reply:
It also says WIP. Sorry for having posted so early in our channel